uber/templates/macros.html

```diff
@@ -82,6 +82,36 @@
 {%- endmacro %}
 
 
+{% macro form_group_scaffolding(
+    label=False,
+    help=False,
+    is_required=False,
+    size=6,
+    form_class='',
+    static_wrap=False) -%}
+    <div class="form-group {{ form_class }}">
+        <label class="col-sm-3 control-label{% if not is_required %} optional-field{% endif %}">
+            {{ label }}
+        </label>
+    <div class="col-sm-{{ size }}">
+        {% if static_wrap %}
+            <div class="form-control-static">
+                {{ caller() }}
+            </div>
+        {% else %}
+            {{ caller() }}
+        {% endif %}
+    </div>
+    {% if help %}
+        <div class="clearfix"></div>
+        <p class="col-sm-{{ size }} col-sm-offset-3 help-block">
```

Comment:
Whoops, I must have missed that detail while pulling the block out. I don't have any strong opinions on the styling. I'll set this to always be col-sm-9.